contracts/proto/faas/v1/faas.proto

```diff
@@ -89,6 +89,23 @@ message ScheduleCron {
   string cron = 1;
 }
 
+// Notification Workers
+enum EventType {
+  All = 0;
+  Created = 1;
+  Deleted = 2;
+}
+
+message BucketNotificationWorker {
+  string bucket = 1;
+	BucketNotificationConfig config = 2;
+}
+
+message BucketNotificationConfig {
+	EventType eventType = 1;
+	string eventFilter = 2;
```

Comment:
change to `event_filter`.

Also Can you add a quick doc comment on what an eventFilter looks like